lametro/views.py

```diff
@@ -680,19 +699,21 @@ def build_form(self, form_kwargs={}):
         form_kwargs['search_corpus'] = 'bills' if self.request.GET.get('search-reports') else 'all'
         form_kwargs['result_type'] = self.request.GET.get('result_type', 'all')
 
-        sqs = SearchQuerySet().facet('bill_type', sort='index')\
-                              .facet('sponsorships', sort='index')\
-                              .facet('legislative_session', sort='index')\
-                              .facet('inferred_status')\
-                              .facet('topics')\
-                              .facet('lines_and_ways')\
-                              .facet('phase')\
-                              .facet('project')\
-                              .facet('metro_location')\
-                              .facet('geo_admin_location')\
-                              .facet('motion_by')\
-                              .facet('significant_date')\
-                              .highlight(**{'hl.fl': 'text,attachment_text'})
+        sqs = SearchQuerySet(
```

Comment:
I don't really like this indentation. Does it also bug you, @fatima3558?

Reply:
I definitely like the new indentation better!